plugin.video.retrospect/addon.xml

```diff
@@ -117,12 +117,17 @@
     <extension point="xbmc.addon.metadata">
         <reuselanguageinvoker>true</reuselanguageinvoker>
         <platform>all</platform>
-        <summary lang="en_GB">Retrospect (previously XBMC Online TV) is a plugin that allows the playback of a lot of TV shows that are available (legally) via the broadcasters websites.</summary>
-        <description lang="en_GB">Retrospect uses the official websites of a lot of different broadcasting companies (mainly Dutch, Belgian, British and Swedish) to make their re-run episodes available on the Kodi platform. More information can be found at www.rieter.net.[CR][CR]Retrospect reports back usage statistics to our servers. This option can be disabled from the add-on settings.</description>
-        <disclaimer lang="en_GB">Retrospect is provided as is. We do not take any responsibility for possible (but highly unlikely) damage caused to your system. Nor do we take responsibility for the content that can be watched via this add-on or any consequences of using this add-on to watch the content using the content provider's platforms.</disclaimer>
-        <license>CC-BY-NC-SA-4.0</license>
+        <summary lang="en_GB">Retrospect enables playback of TV shows legally available online.</summary>
+        <description lang="en_GB">Retrospect uses the official websites of  different broadcasting companies (mainly Dutch, Belgian, British, Norwegian and Swedish) to make their re-run/catchup episodes available on the Kodi platform.[CR][CR]More information can be found at www.rieter.net or the Retrospect wiki at https://github.com/retrospect-addon/plugin.video.retrospect/wiki/.</description>
+        <disclaimer lang="en_GB">Retrospect is provided as is. We do not take any responsibility for possible (but highly unlikely) damage caused to your system. Nor do we take responsibility for the content that can be watched via this add-on or any consequences of using this add-on to watch content by the content providers' platforms.</disclaimer>
+        
+		<summary lang="sv_SE">Retrospect möjliggör att visa TV-program som finns lagligt tillgängliga online.</summary>
```

Comment:
You changed indentation from 4 spaces to single _tab_. Please make the indentation 4 spaces again. This happened on all lines.

Reply:
Fixing.

Reply:
You mean eight spaces, right?